tensilelite/Tensile/SolutionStructs.py

```diff
@@ -3992,14 +4035,16 @@ def calcEpilogueTurns(factorDims: List) -> int:
     if state["EnableMatrixInstruction"] and state["PrefetchLocalRead"] > 0:
       # Multiple = WLR-size / input-size = how many iters could be covered by one WLR ?
       wlrMultiple = state["LocalReadVectorWidth"]//state["MIInputPerThread"]
-      if wlrMultiple == 0:
-        reject(state, "LocalReadVectorWidth %u is less than MIInput" % (state["LocalReadVectorWidth"]))
-        return
+      # NOTE: wlrmultiple can be 0 for new MFMA
+      if not globalParameters["AsmCaps"][tuple(state["ISA"])]["HasMFMA_f8f6f4"] or state["MatrixInstK"] <= 32:
```

Comment:
This check will cause Sparse kernel build failed,  please skip this check when it's generating Sparse kernel.

example:
`not state["ProblemType"]["Sparse"] and (not globalParameters["AsmCaps"][isa]["HasMFMA_f8f6f4"] or state["MatrixInstK"] <= 32):`